src/components/drag-and-drop/draggable-context/draggable-context.js

```diff
@@ -8,37 +8,6 @@ import CustomDragLayer from '../custom-drag-layer';
 import Browser from '../../../utils/helpers/browser';
 import ScrollableParent from '../../../utils/helpers/scrollable-parent';
 
-/**
- * A draggable context component
- *
- * == How to use a draggable context in a component:
- *
- * In your file
- *
- *   import { DraggableContext, WithDrop, WithDrag } from 'carbon-react/lib/components/drag-and-drop'
- *
- * A draggable context is used to define an area in the page where drag and drop can be used on
- * one or more elements (you also need to use WithDrop and WithDrag):
- *
- *   <DraggableContext onDrag={ onItemMoved }>
- *     <ol>
- *       {
- *         items.map((item, index) => {
- *           return (
- *             <WithDrop index={ index }>
- *               <li>
- *                 <WithDrag><span>{ item.content }</span></WithDrag>
- *               </li>
- *             </WithDrop>
- *           );
- *         })
- *       }
- *     </ol>
- *   </DraggableContext>
- *
- * @class DraggableContext
- * @constructor
- */
```

Comment:
I think we need it.
The problem exist only for class components if you wrap it in a functional component, the comment will disappear. (but it is not a good way). We can do fork info addon and fix it as well. I think we need ticket for it and investigate it.

Reply:
I'm happy to add back in, but it doesn't seem to be adding any value at the moment?

Reply:
I don't know if it adding any value at the moment. If you are sure that we can remove it feel free :)